charts/flagger/templates/rbac.yaml

```diff
@@ -276,6 +276,19 @@ rules:
       - /version
     verbs:
       - get
+  - apiGroups:
+      - serving.knative.dev
+    resources:
+      - services
```

Comment:
(just an FYI and this is less common)

You can have a Knative Route and point it to revisions from different knative Configurations. Knative Services are a convenience over it.

I doubt anyone needs _that_ kind of flexibility. Tackling Services rollout will probably cover 99% of use cases.

Reply:
I'll look at this. If it's easy to add I'll do it in this pull request; if not I'll spin it out into another PR